newIDE/app/src/Utils/GDevelopServices/Extension.js

```diff
@@ -43,6 +43,10 @@ export type BehaviorShortHeader = {|
   description: string,
   extensionName: string,
   objectType: string,
+  /**
+   * All required behaviors including transitive ones.
+   */
+  allRequiredBehaviorTypes: Array<string>,
```

Comment:
is the `all` required here? I guess changing this name has a log of impact

Reply:
It's to put emphasis on the transitive aspect.